mix.exs

```diff
@@ -1,26 +1,26 @@
 defmodule StbImage.MixProject do
   use Mix.Project
 
+  @app :stb_image
   @version "0.6.6"
+  @minimum_nif_version "2.14"
   @github_url "https://github.com/elixir-nx/stb_image"
 
   def project do
     [
-      app: :stb_image,
+      app: @app,
       version: @version,
       elixir: "~> 1.12",
       deps: deps(),
       name: "StbImage",
       description: "A tiny image reader/writer library using stb_image as the backend",
       docs: docs(),
       package: package(),
-      make_executable: make_executable(),
-      make_makefile: make_makefile(),
       compilers: [:elixir_make] ++ Mix.compilers(),
       make_precompiler: {:nif, CCPrecompiler},
       make_precompiler_url: "#{@github_url}/releases/download/v#{@version}/@{artefact_filename}",
       make_precompiler_filename: "stb_image_nif",
-      make_precompiler_nif_versions: [versions: ["2.16", "2.17"]]
+      make_precompiler_nif_versions: [versions: [@minimum_nif_version]]
```

Comment:
If this is your minimum version, doesn't it mean you need to precompile with this minimum version?

Reply:
Maybe perhaps we don't change anything. Just leave everything as is? The comfort is actually in knowing that if NIF 2.18 is out, we won't have to release a new version just for it. :) Or you only compile 2.16 from now?

Reply:
I was thinking that we can go as low as 2.14 for Linux and Windows, as for macOS, that should be `2.16` for Apple Silicon and perhaps `2.15` or also `2.14` for Intel Macs because we only uses a few simple NIF functions. 

But we can also just leave everything as is. Not sure how many people are still using old versions like Erlang/OTP 21.

Reply:
So after some experiments, it turned out that nx requries at least Erlang/OTP 24 so we ended up with `2.16` as the minimum NIF version.